addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties

```diff
@@ -181,4 +181,4 @@ channel-type.netatmo.healthindex.state.option.healthy = Gesund
 channel-type.netatmo.healthindex.state.option.fine = Gut
 channel-type.netatmo.healthindex.state.option.fair = Angemessen
 channel-type.netatmo.healthindex.state.option.poor = Schlecht
-channel-type.netatmo.healthindex.state.option.unhealthy = Ungesund
+channel-type.netatmo.healthindex.state.option.unhealthy = Ungesund
```

Comment:
New line missing.
What about changing the `equipmentId` -> `id` in this file and as well as in the `netatmo_fr.properties` file?